### DISCUSSION TITLE
AWS Autoscaling 질문드립니다.

### DISCUSSION THREAD

**Discussion by daecheol93** [tedilabs/fastcampus-devops#10] [General]
개념 강의 중에 강사님께서 트래픽이 많거나 적을때, 자동으로 스케일 업&다운이 된다고, 말씀하셧는데,
스케일 아웃 & 인을 잘못말씀하신게 아닌지.. 스케일 업이된다해서 신기해서 찾아봤는데 찾기가 힘드네요.
인스턴스 CPU가 바뀌려면 중지시키고 바꿔줘야 하는걸로아는데, 자동으로 가능한가요..?

**Comment by posquit0** [comment]
> 개념 강의 중에 강사님께서 트래픽이 많거나 적을때, 자동으로 스케일 업&다운이 된다고, 말씀하셧는데,
> 스케일 아웃 & 인을 잘못말씀하신게 아닌지.. 스케일 업이된다해서 신기해서 찾아봤는데 찾기가 힘드네요.
> 인스턴스 CPU가 바뀌려면 중지시키고 바꿔줘야 하는걸로아는데, 자동으로 가능한가요..?

 안녕하세요! DevOps 초격차 프로그램이 여러 강사분이 강의를 진행하고 있는데요.
해당 파트는 송근일님이 담당하시는 것으로 알고 있으며, 패스트캠퍼스를 통해 문의를 남겨주시면 해당 강사님의 답변을 받아보실 수 있을 것 같습니다.

---
제 개인적인 답변은 ASG가 담당하는 것은 Scale-in/out이 맞습니다. Up/Down은 인스턴스가 재시작되어야 합니다.
다만, Scale-in/out을 Scale-up/down으로 혼용해서 단어를 사용하는 경우도 있으니 참고 바랍니다.
